### ISSUE TITLE
Shortcuts to enable / disable profiles and key blocking

### ISSUE THREAD

**Issue by andyshinn** [Snoothy/UCR#159]
It would be cool if there were ways to start / stop a profile using shortcuts or keybinds. In addition, it would be nice to be able to just temporarily disable blocking globally. It's a feature I make use of now in MouseTovJoy when playing racing games so I can block mouse movements while driving but unblock while in menus.

**Comment by phob** [comment]
Yes disable blocking globally through a keybinding shortcut would be awesome.
